docs/docs/user_guide/style_guide.md

```diff
@@ -129,24 +137,25 @@ enum:
 
 
 ## Root Keys
-Like the Primitive Types, the available root keys in AaC are defined in the definition `Root`. Each field entry in the `Root` definition specifies the root key (the field's name) and a corresponding schema definition (the field type). Additional root keys can be defined by users via an extension to the `roots` definition.
+Like the primitive types, the available root keys in AaC are defined in the definition `Root`. Each field entry in the `Root` definition specifies the root key (the field's name) and a corresponding schema definition (the field type). Additional root keys can be defined by users via an extension to the `Root` definition.
 
-Like primitive types, root keys should follow the snake_case style, but emphasize succint, descriptive names. Example root keys are `import`, `schema`, `model`, etc. If a one-word type isn't descriptive enough for your root key, keeping it succint (e.g. `spring_boot_service`, `cloud_gateway`) is preferable. Long-named root keys, or roots keys that aren't snake_case, will not conform to the standard employed by the Core Specification and first-party plugin making it less obvious to users what the key is.
+Like primitive types, root keys should follow the `snake_case` style, but emphasize succinct, descriptive names. Example root keys are `import`, `schema`, `model`, etc. If a one-word type isn't descriptive enough for your root key, keeping it as succinct as possible (e.g. `spring_boot_service`, `cloud_gateway`) is preferable. Long-named root keys, or roots keys that aren't `snake_case`, will not conform to the standard employed by the Core Specification and first-party plugins and could introduce confusion about what the root key is.
 
 # Organizing Imports
 Each AaC file has a file-wide import declaration definition. Using an `import` definition allows users to reference definitions provided via other user files and can be used to decompose large, complex models into multiple files.
 
-Because `import` decalarations are file-wide, these clauses should be the first clause in AaC files. There is no internal logic that dictates the order of the entries in an AaC file, but maintaining the `import` clause as the first definition in AaC files makes the file-wide dependencies immediately visible and informs readers on the dependencies of file.
+Because `import` decalarations are file-wide, these clauses should be the first clause in AaC files. There is no internal logic that dictates the order of the entries in an AaC file, but maintaining the `import` clause as the first definition in AaC files makes the file-wide dependencies immediately visible and informs readers on the dependencies of file. Additionally, while it's possible to include multiple `import` definitions in a single file, the preferred approach is to list all imported files into a single `import` definition in each file.
```

Comment:
```suggestion
Because `import` decalarations are file-wide, these clauses should be the first clause in AaC files. There is no internal logic that dictates the order of the entries in an AaC file, but maintaining the `import` clause as the first definition in AaC files makes the file-wide dependencies immediately visible and informs readers on the dependencies of file. Additionally, while it's possible to include multiple `import` definitions in a single file, the preferred approach is to list all imported files in a single `import` definition in each file.
```